packages/iTwinUI-react/src/core/Select/Select.tsx

```diff
@@ -6,9 +6,43 @@ import React from 'react';
 import cx from 'classnames';
 import { DropdownMenu } from '../DropdownMenu';
 import { MenuItem } from '../Menu/MenuItem';
-import { PopoverProps, PopoverInstance, CommonProps, useTheme } from '../utils';
+import {
+  PopoverProps,
+  PopoverInstance,
+  CommonProps,
+  useTheme,
+  useOverflow,
+} from '../utils';
 import '@itwin/itwinui-css/css/inputs.css';
 import SvgCaretDownSmall from '@itwin/itwinui-icons-react/cjs/icons/CaretDownSmall';
+import SelectTag from './SelectTag';
+
+const isMultipleEnabled = <T,>(
+  selectedItems: SelectOption<T> | SelectOption<T>[] | undefined,
+  multiple: boolean,
+): selectedItems is SelectOption<T>[] | undefined => {
+  return multiple;
+};
+
+const multipleSelectedItemRendererType = <T,>(
```

Comment:
These should be prefixed by `is`, right? Because it is not a `type`, and we are returning a boolean.

Reply:
I was not sure.. tried to look for guidance over type guard naming, but nothing specific found.
Prefixed with `is`